http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -511,7 +511,6 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
         // Enabled services configs
         raw"cbas.enabled=${config.coaAppConfig.coaServices.contains(Cbas)}",
         raw"cbasUI.enabled=${config.coaAppConfig.coaServices.contains(CbasUI)}",
-        raw"wds.enabled=${config.coaAppConfig.coaServices.contains(Wds)}", // TODO remove after WDS chart migration
```

Comment:
I only just modified this method in [this PR](https://github.com/DataBiosphere/leonardo/pull/3281), and my changes will need to be taken out (leave them only in the wds-specific method) - see down below, around line 521.  There was a bit of rearranging and logic, so let me know if you need help figuring out what all I did and how to undo it.

Reply:
I *think* I did it 👯